### DISCUSSION TITLE
@types/lodash@4.14.187 compilation issue

### DISCUSSION THREAD

**Discussion by LeVraiSylvain** [DefinitelyTyped/DefinitelyTyped#63021] [Issues with a @types package]
Can't compile with version 4.14.187

It works well with 4.14.186

Environment
- node v14.18.1
- tsconfig : in attachment

![image](https://user-images.githubusercontent.com/25437282/199261700-f974942e-585c-46ab-adea-cee75b32fc2b.png)

[a comment by an automated account is not shown]

**Comment by romanshutsman** [comment]
today I have faced the same issue during building an app.

All errors are related to @types/lodash (as we know @types/lodash-es has dependency on @types/lodash ). 
I have such configuration: (can provide more if needed)
```
       "lodash-es": "4.17.15",
      "@types/lodash-es": "^4.17.3",
"@angular/common": "10.1.0",
"typescript": "3.9.6",
```

Any thoughts how to fix?

![image](https://user-images.githubusercontent.com/29098174/199494549-a13e42d9-9183-4a6d-8b26-f330ddf98eff.png)
